package.json

```diff
@@ -17,7 +17,6 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/jest": "^27.0.2",
-    "ethers": "^5.5.1",
```

Comment:
Do we want to add ethers and keccak as devdependencies for the monorepo? Since they are peer deps for the submodules, having them as dev deps makes them available in the env for dev work (like running tests).

Reply:
Exactly, I'm removing it.